dydx-testnet-4/gentx/gentx-01node.json

```diff
@@ -0,0 +1 @@
+{"body":{"messages":[{"@type":"/cosmos.staking.v1beta1.MsgCreateValidator","description":{"moniker":"01node","identity":"7BDD4C2E94392626","website":"https://01node.com","security_contact":"secops@01node.com","details":"Securely validating one block at a time since 2019"},"commission":{"rate":"1.000000000000000000","max_rate":"1.000000000000000000","max_change_rate":"0.010000000000000000"},"min_self_delegation":"1","delegator_address":"dydx1jvvmq2sjck6ztr236z7jva3cgv52msetslrnzr","validator_address":"dydxvaloper1jvvmq2sjck6ztr236z7jva3cgv52mset4p49fj","pubkey":{"@type":"/cosmos.crypto.ed25519.PubKey","key":"g6rIeRa+u3WTgCUweFzfBhhLfHx8qR3J+vA0GIj67BM="},"value":{"denom":"dv4tnt","amount":"50000000000"}}],"memo":"a84193c5411fed5d4accc7e08204c06adf00c972@0.0.0.0:26656","timeout_height":"0","extension_options":[],"non_critical_extension_options":[]},"auth_info":{"signer_infos":[{"public_key":{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"AsilcP7iTbbQpRAODmiOrEdVk9TIiZ3i8rEHc31hQu1Y"},"mode_info":{"single":{"mode":"SIGN_MODE_DIRECT"}},"sequence":"0"}],"fee":{"amount":[],"gas_limit":"200000","payer":"","granter":""},"tip":null},"signatures":["lnTl/RNUa+380kcDcaZ4P+L2+6oOzw7ucOmEwo/C6lAoFktXeIpZ8ukwQ1sk5CPp70jZ8Ou5bnJpACxiYvVkUA=="]}
```

Comment:
hi, @JiuKelo
the [Prepare for Genesis](https://v4-teacher.vercel.app/validators/preparing_for_genesis) guide is a little different this time, the test token is adv4tnt and the self-delegation amount is 500000000000000000000000adv4tnt

Reply:
fixed